### PR TITLE
Update BO theme/default webpack to v4.41 and its dependencies

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | We wants to update webpack on our themes so we will be able to merge them into only one package.json, I updated webpack to v4.41 and its linked dependencies. 
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| How to test?  | What you need to test is the fact that legacy pages are the same as before this PR, nothing should change after building the assets of admin-dev/theme/default !

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17426)

### CODE REVIEW

**Comment by Progi1984** [comment]
@NeOMakinG Please rebase your branch

File: admin-dev/themes/default/package.json [hunk 1]
```diff
@@ -1,38 +1,39 @@
...
+    "node-sass": "^4.13.1",
+    "open-sans-fonts": "^1.5.1",
+    "optimize-css-assets-webpack-plugin": "^5.0.0",
+    "path": "^0.12.7",
+    "postcss-cssnext": "^3.1.0",
+    "postcss-loader": "^3.0.0",
+    "sass-loader": "^8.0.2",
+    "style-loader": "^1.1.3",
+    "uglify-save-license": "^0.4.1",
+    "uglifyjs-webpack-plugin": "^2.2.0",
```
Comment (Progi1984) [inline admin-dev/themes/default/package.json:32]: @NeOMakinG Have you checked the license for the package `uglifyjs-webpack-plugin` ?

Comment (NeOMakinG) [reply admin-dev/themes/default/package.json:32]: Was already used before but was centralized inside webpack, it's MIT

Comment (Progi1984) [reply admin-dev/themes/default/package.json:32]: :ok:

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by atomiix** [comment]
Thanks @NeOMakinG ! 🎉

**Pull request merged**